ddr/tools/getmacros

```diff
@@ -30,6 +30,7 @@ default_regex="@ddr_namespace: *default"
 map_to_type_regex="@ddr_namespace: *map_to_type="
 flag_define_regex="^ *# *define +(${id}) *($|//|/\*)"
 flag_undef_regex="^ *# *undef +(${id}) *($|//|/\*)"
+flag_undef_regex_comment="^/\* # *undef +(${id}) \*/"
```

Comment:
We want to be stricter here: don't accept spaces between `#` and `undef`.

Reply:
Alright, should the other regexes be changed too?

Reply:
No, the others are correct as is - they recognize patterns in regular code.